readme.md

```diff
@@ -25,6 +25,23 @@ Tests
 
 To run the tests and linter locally, run `npm test`.
 
+A Day in the Life of an AMP Article
+---
+
+1. Google crawls FT.com and finds an article, `/content/ffffffff-ffff-ffff-ffff-ffffffffffff`
+2. The article has a `<link rel="amphtml">`, with href pointing to `amp.ft.com/content/ffffffff-ffff-ffff-ffff-ffffffffffff`
+3. The [amp-page.js](server/controllers/amp-page.js) controller runs
+    1. It calls [get-article.js](server/lib/article/get-article.js) to fetch the content from Elastic Search
+    2. It runs various [transforms](server/lib/transforms) on the [article data](server/lib/transforms/article.js) and [body HTML(server/lib/transforms/body.js)
```

Comment:
borken md on the body HTML link

(2 could probably also use a "...which Google then crawls" suffix)